mission/landmarks/src/landmarks.cpp

```diff
@@ -0,0 +1,45 @@
+#include "ros/ros.h"
+#include "geometry_msgs/Point.h"
+#include <vortex_msgs/ObjectPosition.h>
+#include <map>
+#include <string>
+
+class Landmarks{
```

Comment:
Doxygen style comments would usually go in a .h file, but since landmarks.cpp does not have a header, the comments should be put here.